src/html_support_files/odoc.css

```diff
@@ -725,7 +725,13 @@ td.def-doc *:first-child {
 /* Alert emoji */
 
 .alert::before, .deprecated::before {
-  content: '⚠️ ';
+  content: '⚠️ ' / '';
```

Comment:
Just informational, the ` / ''` here is to set the alt text to an empty string which tells the screen reader to not read out the emoji because it is strictly decorative and redundant for spoken text.

Reply:
Very interesting, thanks!